### PR TITLE
chore: Update sub-module url

### PR DESCRIPTION
**Pull request by rish691**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -8,21 +8,21 @@
...
 	branch = main
 [submodule "query-service"]
 	path = query-service
-	url = https://github.com/hypertrace/query-service.git
+	url = git@github.com:hypertrace/query-service.git
 	branch = main
 [submodule "hypertrace-graphql"]
 	path = hypertrace-graphql
-	url = https://github.com/hypertrace/hypertrace-graphql.git
+	url = git@github.com:hypertrace/hypertrace-graphql.git
```
Comment (kotharironak) [inline .gitmodules:19]: we have done `http` based as everyone who tries out might not have created `ssh` key.

**Review by kotharironak — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 1 above)
Comment (kotharironak) [reply .gitmodules:19]: change this with `http` - git@github.com:hypertrace/config-bootstrapper.git

**Comment by JBAhire** [comment]
We made this change earlier here: https://github.com/hypertrace/hypertrace-service/pull/10 to ensure users not in our org won't face authentication problems

File: .gitmodules (hunk 1 above)
Comment (rish691) [reply .gitmodules:19]: Saw this https://github.com/hypertrace/hypertrace-service/pull/10/files 
Will move back to https

**Pull request closed without merging**